### PR TITLE
Update the package with latest runtime API changes for type reference

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

Related to https://github.com/ballerina-platform/ballerina-standard-library/issues/3517
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec
- [ ] Checked native-image compatibility

### CODE REVIEW

File: ballerina/Dependencies.toml [hunk 1]
```diff
@@ -28,7 +28,7 @@ modules = [
 [[package]]
 org = "ballerina"
 name = "io"
-version = "1.3.1"
+version = "1.4.0"
```
Comment (HindujaB) [inline ballerina/Dependencies.toml:31]: Shall we change the distribution in here as well?
https://github.com/ballerina-platform/module-ballerina-file/blob/5c002f3b73d4cb972fa52b2d727d80083f4692cf/build-config/resources/Ballerina.toml#L10

File: ballerina/Dependencies.toml [hunk 2]
```diff
@@ -28,7 +28,7 @@ modules = [
 [[package]]
 org = "ballerina"
 name = "io"
-version = "1.3.1"
```
Comment (HindujaB) [inline ballerina/Dependencies.toml:31]: Need to update the version and jar names in here as well
https://github.com/ballerina-platform/module-ballerina-file/blob/master/ballerina/Ballerina.toml

File: ballerina/Dependencies.toml (hunk 1 above)
Comment (kalaiyarasiganeshalingam) [reply ballerina/Dependencies.toml:31]: Updated in [ba252f9](https://github.com/ballerina-platform/module-ballerina-file/pull/475/commits/ba252f947b23fd76d69c8220f08fc165d4d62924)

File: ballerina/Dependencies.toml (hunk 2 above)
Comment (kalaiyarasiganeshalingam) [reply ballerina/Dependencies.toml:31]: Updated in [ba252f9](https://github.com/ballerina-platform/module-ballerina-file/pull/475/commits/ba252f947b23fd76d69c8220f08fc165d4d62924)

**Review by HindujaB — APPROVED** [review]
LGTM

**Pull request merged**